dist/main.js

```diff
@@ -0,0 +1,31 @@
+!function(c){var e={};function n(a){if(e[a])return e[a].exports;var t=e[a]={i:a,l:!1,exports:{}};return c[a].call(t.exports,t,t.exports,n),t.l=!0,t.exports}n.m=c,n.c=e,n.d=function(c,e,a){n.o(c,e)||Object.defineProperty(c,e,{enumerable:!0,get:a})},n.r=function(c){"undefined"!=typeof Symbol&&Symbol.toStringTag&&Object.defineProperty(c,Symbol.toStringTag,{value:"Module"}),Object.defineProperty(c,"__esModule",{value:!0})},n.t=function(c,e){if(1&e&&(c=n(c)),8&e)return c;if(4&e&&"object"==typeof c&&c&&c.__esModule)return c;var a=Object.create(null);if(n.r(a),Object.defineProperty(a,"default",{enumerable:!0,value:c}),2&e&&"string"!=typeof c)for(var t in c)n.d(a,t,function(e){return c[e]}.bind(null,t));return a},n.n=function(c){var e=c&&c.__esModule?function(){return c.default}:function(){return c};return n.d(e,"a",e),e},n.o=function(c,e){return Object.prototype.hasOwnProperty.call(c,e)},n.p="",n(n.s=22)}([function(c,e,n){"use strict";c.exports=n(9)},function(c,e,n){var a=n(16);"string"==typeof a&&(a=[[c.i,a,""]]);var t={hmr:!0,transform:void 0,insertInto:void 0};n(18)(a,t);a.locals&&(c.exports=a.locals)},function(c,e,n){c.exports=n(20)()},function(c,e){var n;n=function(){return this}();try{n=n||new Function("return this")()}catch(c){"object"==typeof window&&(n=window)}c.exports=n},function(c,e,n){"use strict";var a=function(){};c.exports=a},function(c,e,n){"use strict";
```

Comment:
Totally forgot to tell you, can you please git ignore everythin under `dist/`, it is the equivalent of the built website 📦 🔥